specification.md

```diff
@@ -506,11 +506,11 @@ See paragraph 5.1 of [WCA Competition Requirements Policy](https://www.worldcube
 
 Represents a competitor result in a single round.
 
-| Attribute | Type | Description |
-| --- | --- | --- |
-| `personId` | `Integer` | The corresponding person `registrantId`. |
-| `ranking` | `Integer\|null` | The ranking in this round. May be `null` if the result is empty (yet to be entered). |
-| `attempts` | [`Attempt`](#attempt) | List of attempt results the competitor got. If there are fewer attempts than expected, the rest is considered skipped (effectively `0`). |
+| Attribute | Type                              | Description |
+| --- |-----------------------------------| --- |
+| `personId` | `Integer`                         | The corresponding person `registrantId`. |
+| `ranking` | `Integer\|null`                   | The ranking in this round. May be `null` if the result is empty (yet to be entered). |
+| `attempts` | [`[Attempt]`](#attempt)           | List of attempt results the competitor got. If there are fewer attempts than expected, the rest is considered skipped (effectively `0`). |
```

Comment:
I would keep the current formatting for consistency, it's also easier to maintain :)

```suggestion
| Attribute | Type                              | Description |
| --- | --- | --- |
| `personId` | `Integer` | The corresponding person `registrantId`. |
| `ranking` | `Integer\|null` | The ranking in this round. May be `null` if the result is empty (yet to be entered). |
| `attempts` | [`[Attempt]`](#attempt) | List of attempt results the competitor got. If there are fewer attempts than expected, the rest is considered skipped (effectively `0`). |
```